gdx/src/com/badlogic/gdx/graphics/VertexAttribute.java

```diff
@@ -184,11 +147,27 @@ public int getKey () {
 		return (usageIndex << 8) + (unit & 0xFF);
 	}
 	
+	/** @return How many bytes this attribute uses. */
+	public int getNumBytes () {

```

Comment:
I'll go with getSizeInBytes to avoid confusion with VertexAttributes.getSize() which returns number of components.